ext/native-decls/RegisterCommand.md

```diff
@@ -19,24 +19,31 @@ The restricted bool is not used on the client side. Permissions can only be chec
 ![](https://i.imgur.com/TaCnG09.png)
 
 
-## Parameters
+## Required Arguments
 * **commandName**: The command you want to register.
 * **handler**: A handler function that gets called whenever the command is executed.
+
+## Optional Arguments
 * **restricted**: If this is a server command and you set this to true, then players will need the command.yourCommandName ace permission to execute this command.
 
+## Handler function parameters
+* **playerSource**: The player font which executed the command.
```

Comment:
Why change `source` -> `playerSource` when `source` is consistent across all of the documentation?

Reply:
'The player font which'?

Reply:
'source' Should be used in cases when it is not in Handler function parameters

Reply:
> 'source' Should be used in cases when it is not in Handler function parameters

Doesn't make much sense, as it is still their source?

Reply:
> > 'source' Should be used in cases when it is not in Handler function parameters
> 
> Doesn't make much sense, as it is still their source?

I'll use a function from the MTA: SA platform as an example, look at: https://wiki.multitheftauto.com/wiki/OnMarkerHit